File: CloudApi/src/main/java/com/proxy/service/api/log/LogInit.java
package com.proxy.service.api.log;

import com.proxy.service.api.log.tree.DebugTree;
import com.proxy.service.api.log.tree.ReleaseTree;
import com.proxy.service.api.log.tree.TreeGroup;

/**
 * @author: cangHX
 * @data: 2024/4/28 18:53
 * @desc:
 */
public class LogInit {

    public static void setIsDebug(boolean isDebug) {
        if (isDebug) {
            TreeGroup.plant(new DebugTree());
        } else {
            TreeGroup.plant(new ReleaseTree());
        }
    }
}

File: CloudApi/src/main/java/com/proxy/service/api/log/LogInit.kt
package com.proxy.service.api.log

import com.proxy.service.api.log.tree.DebugTree
import com.proxy.service.api.log.tree.ReleaseTree
import com.proxy.service.api.log.tree.TreeGroup

/**
 * @author: cangHX
 * @data: 2024/4/28 18:53
 * @desc:
 */
object LogInit {

    fun setIsDebug(isDebug: Boolean) {
        val tree = if (isDebug) {
            DebugTree()
        } else {
            ReleaseTree()
        }
        TreeGroup.plant(tree)
    }
}